javascript/javascript1/week1/script.js

```diff
@@ -80,3 +86,38 @@ if (juliaHouse <= 1000000) {
   let difference2 = juliaHouse - 1000000;
   console.log(`Julia you are buying ${difference2} cheaper than market price`);
 }
+
+//Ez Namey (Startup name generator)
+
+let firstWords = [
+  "Easy",
+  "Awesome",
+  "Corporate",
+  "Hyper",
+  "Rocket",
+  "Atom",
+  "Advanced",
+  "Technology",
+  "Active",
+  "Fast",
+];
+let secondWords = [
+  "Solutions",
+  "Thinks",
+  "Gadgets",
+  "Wisdom",
+  "Super",
+  "Fresh",
+  "Network",
+  "Space",
+  "Palace",
+  "Platform",
+];
+let startupName = null;
+const randomNumber = Math.floor(Math.random() * 10);
+
+function startupNameGenerator() {
+  let startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
+  return startupName;
+}
+console.log(startupNameGenerator(`The startup: ${startupName}`));
```

Comment:
This is very good to see that you have used function for this exercise, But I don't think using function is the best solution for this exercise. 
I would use a variable which returns a random name each time, and that is what the exercise wants from us. 
The main reason to use function is to make reusable solutions, like what you did for housyPricy exercise, but here you didn't define parameters for `startupNameGenerator()` which makes it not reusable. You have also used arguments when you called your function, which is a mistake. If you want the function to print something out you should define it in the function body. Also in the homework says that you need to count the length of the start up name.